---
title: "<b>1\\.</b> 
Kirilenko, B.M., Munegowda, C., Osipova, E., Jebb, D., Sharma, V., Blumer, M.,
Morales, A., Ahmed, A.-W., <u>Kontopoulos, D.-G.</u>, Hilgers, L., 
Zoonomia Consortium, and Hiller, M.<sup><a title='Corresponding author'>âœ‰</a></sup> (2022). **TOGA 
integrates gene annotation with orthology inference at scale.** bioRxiv 2022.09.08.507143."
collection: publications
permalink: /publication/01_TOGA
pubtype: 'preprint'
excerpt: '
  [<span class="underline-on-hover" style="color:#2ea9d8">Abstract</span>](../publication/01_TOGA)
\| [<span class="underline-on-hover" style="color:#2ea9d8">Full text</span>](https://doi.org/10.1101/2022.09.08.507143)
'
---

<br><center><img src="../images/publications/TOGA.png"></center>

### Abstract:

<p style='text-align: justify;'>
Annotating coding genes and inferring orthologs are two classical 
challenges in genomics and evolutionary biology that have traditionally 
been approached separately, which limits scalability. We present TOGA, 
the first method that integrates gene annotation and orthology inference. 
TOGA implements a novel paradigm to infer orthologous genes, improves 
ortholog detection and annotation completeness compared to state-of-the-art 
methods, and handles even highly-fragmented assemblies. TOGA scales to 
hundreds of genomes, which we demonstrate by applying it to 488 
placental mammal and 308 bird assemblies, creating the largest 
comparative gene resources so far. Additionally, TOGA detects gene 
losses, enables selection screens, and automatically provides a 
superior measure of mammalian genome quality. Together, TOGA is a 
powerful and scalable method to annotate and compare genes in the genomic era.

</p>

[<span class="underline-on-hover" style="color:#2ea9d8">Full text</span>](https://doi.org/10.1101/2022.09.08.507143)
