---
title: "<b>13\\.</b> 
Kirilenko, B.M., Munegowda, C., Osipova, E., Jebb, D., Sharma, V., Blumer, M.,
Morales, A.E., Ahmed, A.-W., <u>Kontopoulos, D.-G.</u>, Hilgers, L., 
Lindblad-Toh, K., Karlsson, E.K., Zoonomia Consortium, and Hiller, M.<sup><a title='Corresponding author'>âœ‰</a></sup> (2023). **Integrating 
gene annotation with orthology inference at scale.** Science 380(6643):eabn3107. <img src='../images/open_access.png'>"
collection: publications
permalink: /publication/13_TOGA
pubtype: 'peer-reviewed'
excerpt: '
  [<span class="underline-on-hover" style="color:#2ea9d8">Abstract</span>](../publication/13_TOGA)
\| [<span class="underline-on-hover" style="color:#2ea9d8">Full text</span>](https://doi.org/10.1126/science.abn3107)
\| [<span class="underline-on-hover" style="color:#2ea9d8">Preprint</span>](https://doi.org/10.1101/2022.09.08.507143)
\| [<span class="underline-on-hover" style="color:#2ea9d8">Source code</span>](https://github.com/hillerlab/TOGA)
\| [<span class="underline-on-hover" style="color:#2ea9d8"><img src="../images/bibtex.svg">citation</span>](../bibtex/13_TOGA.bib)
\|
<span class="__dimensions_badge_embed__" data-doi="10.1126/science.abn3107" data-hide-zero-citations="true" data-legend="never" data-style="large_rectangle" style="display: inline;"></span>
<br>
'
---

{{ page.title }}<br>
<br><center><img src="../images/publications/TOGA.png"></center>

### Abstract:

<p style='text-align: justify;'>
Annotating coding genes and inferring orthologs are two classical 
challenges in genomics and evolutionary biology that have traditionally 
been approached separately, limiting scalability. We present TOGA (Tool to infer Orthologs from Genome Alignments), 
a method that integrates structural gene annotation and orthology inference. 
TOGA implements a different paradigm to infer orthologous loci, improves 
ortholog detection and annotation of conserved genes compared with state-of-the-art 
methods, and handles even highly fragmented assemblies. TOGA scales to 
hundreds of genomes, which we demonstrate by applying it to 488 
placental mammal and 501 bird assemblies, creating the largest 
comparative gene resources so far. Additionally, TOGA detects gene 
losses, enables selection screens, and automatically provides a 
superior measure of mammalian genome quality. TOGA is a 
powerful and scalable method to annotate and compare genes in the genomic era.

</p>

[<span class="underline-on-hover" style="color:#2ea9d8">Full text</span>](https://doi.org/10.1126/science.abn3107)
\| [<span class="underline-on-hover" style="color:#2ea9d8">Preprint</span>](https://doi.org/10.1101/2022.09.08.507143)
\| [<span class="underline-on-hover" style="color:#2ea9d8">Source code</span>](https://github.com/hillerlab/TOGA)
\| [<span class="underline-on-hover" style="color:#2ea9d8"><img src="../images/bibtex.svg">citation</span>](../bibtex/13_TOGA.bib)
